src/TimeField.php

```diff
@@ -37,7 +37,7 @@ public function __construct($name, $attribute = null, $resolveCallback = null)
      *
      * @return $this
      */
-    public function nullable()
+    public function nullable($nullable = true, $value = null )
```

Comment:
Shouldn't it be `return $this->withMeta(['nullable' => $nullable]);`. What is the $value used for in Nova 2.0?